src/test/java/emissary/test/core/TestExtractionTest.java

```diff
@@ -8,17 +10,17 @@
 import org.jdom2.Element;
 import org.jdom2.JDOMException;
 import org.jdom2.input.SAXBuilder;
-import org.junit.Assert;
-import org.junit.Test;
+import org.junit.jupiter.api.Assertions;
```

Comment:
Replace with a static import of `org.junit.jupiter.api.Assertions.assertNotNull`